### PR TITLE
update winter olympics and five great reads banners

### PR DESCRIPTION
**Pull request by dblatcher**
## What does this change?
 - add new banner for five great reads
 - add new banner for winter olympics 2022 (left the banner for 2018 in place)

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

Tested locally:
Screenshot 2022-01-20 at 10 15 19

### CODE REVIEW

**Comment by dblatcher** [comment]
[Add/change banners for composer newsletters](https://trello.com/c/dvBE6ySI/524-add-change-banners-for-composer-newsletters)

File: common/app/model/EmailAddons.scala [hunk 1]
```diff
@@ -321,6 +321,11 @@ case object WinterOlympics2018 extends FrontEmailMetadata {
   override val banner = Some("winter-olympics-2018.png")
 }
 
+case object WinterOlympics2022 extends FrontEmailMetadata {
+  val name = "Winter Olympics 2022"
+  override val banner = Some("winter-olympics-2022.png")
+}
```
Comment (OllysCoding) [inline common/app/model/EmailAddons.scala:327]: Does this object need to be included in the `frontEmails` sequence at the bottom of the file?

**Review by oliverlloyd — COMMENTED** [review]
👋 David, thanks for this PR. Seconding @OllysCoding 's comment about if the object should be used somewhere or not?

And also, would ie be possible to reduce the size of these images a bit? Every kB counts!

File: common/app/model/EmailAddons.scala (hunk 1 above)
Comment (dblatcher) [reply common/app/model/EmailAddons.scala:327]: Yes, thanks. I've added that

**Comment by dblatcher** [comment]
> 
I've just added smaller versions of the image files - sorry, I thought the ones I'd been give were already 'final'. They are still on the large side compared to some of the others in the folder, but in the right ballpark now.

**Comment by dblatcher** [comment]
We had some updates on our instructions - the winter olympics banner was actually supposed to be Article based. This has been changed now.

**Review by OllysCoding — APPROVED** [review]
+1 LGTM

**Review by oliverlloyd — APPROVED** [review]
Great work!

**Pull request merged**